### PR TITLE
Bazel tweaks

### PR DESCRIPTION
**Pull request by RussTedrake**
start using bazel instead of cmake

### CODE REVIEW

**Comment by RussTedrake** [comment]
ps - @jamiesnape , i assume you're aware that circleci has been broken all yesterday?

File: .bazelrc [hunk 1]
```diff
@@ -33,6 +33,16 @@
 build --test_env=LC_CTYPE=en_US.UTF-8
 build --test_env=MPLBACKEND=Template
 
+# Location of the Gurobi license key file, typically named "gurobi.lic".
+# Setting this --test_env for all configurations is deliberate to improve
+# remote caching performance.
+test --test_env=GRB_LICENSE_FILE
```
Comment (jamiesnape) [inline .bazelrc]: This does not make sense for the binary workflow.

File: .bazelrc [hunk 2]
```diff
@@ -33,6 +33,16 @@
...
+
+# Location of the MOSEK license file, typically named "mosek.lic". Setting
+# this --test_env for all configurations is deliberate to improve remote
+# caching performance.
+test --test_env=MOSEKLM_LICENSE_FILE
```
Comment (jamiesnape) [inline .bazelrc]: As above.

File: environ.bzl [hunk 3]
```diff
@@ -0,0 +1,70 @@
+# -*- mode: python -*-
```
Comment (jamiesnape) [inline environ.bzl]: Move to `//tools` for now, though I may move/replace it once I have support (almost complete) for downloading both Linux and macOS binary package automatically.

File: README.md [hunk 4]
```diff
@@ -39,6 +40,13 @@ $ make
 $ ctest .
 ```
 
+To run the unit tests (bazel)
```
Comment (jamiesnape) [inline README.md]: `To run the unit tests using Bazel`

File: README.md [hunk 5]
```diff
@@ -29,8 +29,9 @@ https://websitebeaver.com/set-up-localhost-on-macos-high-sierra-apache-mysql-and
 and just pointed by root doc directory directly at my underactuated checkout.
 
 
-To run the unit tests
----------------------
+To run the unit tests (cmake)
+-----------------------------
+CircleCI currently runs the tests using cmake; this will soon be removed in favor of the bazel workflow below.
```
Comment (jamiesnape) [inline README.md]: `cmake` -> `CMake`,  `bazel` -> `Bazel`

File: README.md [hunk 6]
```diff
@@ -29,8 +29,9 @@ https://websitebeaver.com/set-up-localhost-on-macos-high-sierra-apache-mysql-and
 and just pointed by root doc directory directly at my underactuated checkout.
 
 
-To run the unit tests
----------------------
+To run the unit tests (cmake)
```
Comment (jamiesnape) [inline README.md]: `To run the unit tests using CMake`

File: README.md [hunk 7]
```diff
@@ -39,6 +40,13 @@ $ make
 $ ctest .
 ```
 
+To run the unit tests (bazel)
+-----------------------------
+Github actions runs the tests using bazel.  This is preferred, and will replace the cmake workflow once it has complete coverage.  See #245.
```
Comment (jamiesnape) [inline README.md]: `Github` -> `GitHub`, `bazel` -> `Bazel`, `cmake` -> `CMake`, See `#245` -> `[#245](https://github.com/RussTedrake/underactuated/issues/245)`.

File: WORKSPACE.bazel [hunk 8]
```diff
@@ -56,11 +56,17 @@ load("@pip//:requirements.bzl", "pip_install")
 
 pip_install()
 
+# Set the INSTALLED_DRAKE_DIR environment variable to the correct path; if 
+# INSTALLED_DRAKE_DIR is not set it will look in "/opt/drake".
+load("//:environ.bzl", "environ_repository")
```
Comment (jamiesnape) [inline WORKSPACE.bazel]: This would be updated to `//tools:environ.bzl`

**Comment by jamiesnape** [comment]
> ps - @jamiesnape , i assume you're aware that circleci has been broken all yesterday?

Looking into it now.

**Comment by jamiesnape** [comment]
We have a cycle now in the WORKSPACE file. Not sure why yet. Maybe split the WORKSPACE-related changes from the documentation into a different PR?

File: .bazelrc (hunk 1 above)
Comment (RussTedrake) [reply .bazelrc]: added `build --action_env=GRB_LICENSE_FILE` also.  i assume that was your intention?

File: .bazelrc (hunk 2 above)
Comment (RussTedrake) [reply .bazelrc]: same.

File: environ.bzl (hunk 3 above)
Comment (RussTedrake) [reply environ.bzl]: Done.  (i took you literally, and put it in `//tools`.  not sure if you meant `//tools/rt`.

**Pull request merged**